napari/layers/labels/_labels_constants.py

```diff
@@ -35,4 +35,17 @@ class Mode(StringEnum):
     ERASE = auto()
 
 
+class LabelColorMode(StringEnum):
+    """
+    LabelColorMode: Labelling Color setting mode.
+
+    RANDOM (default) allows color to be set via a hash function with a seed.
```

Comment:
I liked the suggestion from @kevinyamauchi to make this `AUTO` instead of `RANDOM` I think that is a bit more accurate.